src/frontend/src/components/RecordSearch/RecordSearch.test.tsx

```diff
@@ -12,6 +14,11 @@ jest.mock("../../service/cookie-service", () => ({
   hasOeciToken: () => mockHasOeciToeken(),
 }));
 
+const DemoOnComponent = () => {
+  useAppDispatch()(startDemo());
```

Comment:
Just for future reference, I prefer to have functions named even for small components like this.